Resolve discussion about EKU requirements (#69)
creator-assertions/identity-assertion: docs/modules/ROOT/pages/index.adoc

* Any reference to the _claim_ MUST be replaced with the CBOR serialization of the `referenced_assertions` field from the *<<_identity_assertion,identity assertion>>.*
* Any reference to the _claim generator_ MUST be replaced with the _<<_actor,actor>>_ whose _X.509 certificate_ is being used for this assertion.

> * The validator SHALL maintain a trust list which SHALL be independent from the trust list used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”] The trust list for identity assertion signers need not include the C2PA Trust List as defined in that section.

Reviewer: ```suggestion
* The validator SHALL maintain one or more trust lists which do not need to be the same as the trust lists used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”]. 
```

* Any reference to the _claim_ MUST be replaced with the CBOR serialization of the `referenced_assertions` field from the *<<_identity_assertion,identity assertion>>.*
* Any reference to the _claim generator_ MUST be replaced with the _<<_actor,actor>>_ whose _X.509 certificate_ is being used for this assertion.

> * The validator SHALL maintain a trust list which SHALL be independent from the trust list used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”] The trust list for identity assertion signers need not include the C2PA Trust List.

Reviewer 2: Where is the C2PA Trust List specified? (should be good to add a pointer in the spec too)

Author: @christianpaquin this is current work of the C2PA Trust List Task Force, which hasn't yet (as far as I know) published any specifications specifics to the trust list. I don't think there's more to link at this time to than §14.4.1, which says there is (or should be) such a trust list. I added another "as defined in that specification" to that section; not sure what more I can do until C2PA publishes more.
